workbench-core/datasets/src/dataSet.ts

```diff
@@ -19,6 +19,21 @@ export interface DataSet {
    */
   createdAt?: string;
 
+  /**
+   * (optional) a description of the dataset
+   */
+  description?: string;
+
+  /**
+   * (optional) the owner of the dataset
+   */
+  owner?: string;
+
+  /**
+   * (optional) the type of the dataset
+   */
+  type?: string;
```

Comment:
Should we use enum here?

Reply:
We dont want to restrict type to a defined enum. This still works though if SWB wants to use an enum internally:
```ts
enum TestEnum {
  STRING_ONE = 'one',
  STRING_TWO = 'two
}

const testString: string = TestEnum.STRING_ONE;
```